antenna-documentation/src/site/markdown/sw360-data-model.md.vm

```diff
@@ -0,0 +1,69 @@
+In the following section there is an outline what data of SW360 ${docNameCap} maps.
+
+#[[###]]# Projects
+ When the workflow steps of ${docNameCap} are run they create a SW360 project that is the current executed project.
+ The coordinates of this project can be set in the [`config.xml`](config-configuration.html).
+ 
+  | Variable Name         | Meaning                       |
+  |-----------------------|-------------------------------|
+  | Type | the name of the component |
+  | Name | the name of the project |
+  | Version | Current version of the project |
+  | Project Type | What type of project. E.g. "Product" |
+  | Description | Description of the project. E.g what it does and who is responsible for it. |
+  | Created On| Date of creation in SW360 instance |
+  | Business Unit | Business Unit |
+  | Clearing Team | Clearing Team |
+  | Visibility | Which roles can see the project |
+  | ReleaseIDToUsage | The release ID to the project usage |
+  | External Ids | Additional information can be put here in a `Map<String,String>` format |
+
+#[[###]]# Licenses
+ All licenses are created separately from the components and releases.
+ Enriching and updating is also done separately.
+
+  | Variable Name         | Meaning                       |
+  |-----------------------|-------------------------------|
+  | Text | The text of the license |
+  | Short Name | The short name of the license. E.g. `Apache-2.0` |
+  | Full Name | The full, long name of the license. E.g. `Apache Software License 2.0` |
+ 
+#[[###]]# Components
+ One component is an individual component with no version.
+ The individual versions of a component are called "releases" and are saved as a different object.
+ Each component has a list of its releases.
+
+ | Variable Name         | Meaning                       |
+ |-----------------------|-------------------------------|
+ | Name | the name of the component E.g. Junit:Junit |
+ | Component Type| Can be `INTERNAL`, meaning it is proprietary, or `OSS`, meaning it is open source. |
+ | Created On| Date of creation in SW360 instance |
+ | Homepage| Link to the homepage of the OSS Component |
+ 
+#[[###]]# Releases
+ An individual release is a release of a component, meaning it always has a version and always has a component it belongs to.
+ All license and copyright information can be found here.
+
+  | Variable Name         | Meaning                       |
+  |-----------------------|-------------------------------|
+  | Component ID | ID of the component this release belongs to. E.g. The ID of the Junit component |
+  | Name | Name of the release, which should equal the component name. E.g. `junit:junit` |
+  | Version | Version of the release. E.g 4.8.1, which would come to the component `junit:junit:3.8.1` |
+  | CPE ID | Common Platform Enumeration ID |
+  | Download Url | The link where the source code can be found. Similar as a "Source Url" |
```

Comment:
`Download Url` isn't the URL to the (compiled) artifact?

Reply:
The url to the compiled, binary artifact, really? I see this as the link to the potentially curated sources stored in a safe accessible place, e.g., internally. For compliance reason, I need access to the sources corresponding to the artifact, the real used dependency is by coordinates or hash codes uniquely defined.

Reply:
That was just purely based on the name. If I see `Download Url` of a release, I expect to find there the release instead of the sources or curated sources.

Maybe the name is not descriptive enough?

Reply:
Right, but being not descriptive helps using the tool in a different context :-). I see no big value in storing the url to the binary dependency, since it is typically retrieved from someplace else and the entry to the process. But a link to the "official" uses is something needed for processing.

Reply:
Would this be more adequate?: 
`  | Download Url | Link of a source archive, typically from a server, which is under the organizations control, e.g. an internal Nexus instance. |`

Reply:
Proposal: Link to the source archive. E.g., used to aggregate the sources of the used OSS components for the FOSS compliance bundle